docs/Getting Started/Example Apps/index.md

```diff
@@ -123,3 +127,40 @@ The [Hello Sdl JavaEE](https://github.com/smartdevicelink/sdl_java_suite/tree/ma
 To connect the sample app to the infotainment system, please follow the instructions in the [Connecting to an Infotainment System](Getting Started/Connecting to an Infotainment System) guide.
 !@
 
+@![javascript]
+## Connecting to an Infotainment System
+### Connecting as a WebSocket Client
+For vanilla JavaScript SDL apps, connecting as a WebSocket client requires a version of SDL Core that can accept incoming WebSocket connections (at least v6.1.0). If you are using Manticore to test your app, note that it currently does not support WebSocket connections.
+A workaround to this limitation is to use a proxy program for your app to connect with which modifies the incoming WebSocket connection into a TCP connection. The proxy program then connects to SDL Core on the app's behalf and passes through your transport data. A Java program that does exactly this is available in the [repository's JavaScript example folder](https://github.com/smartdevicelink/sdl_javascript_suite/blob/master/examples/js/hello-sdl) called `proxy.jar`. Check the example app's `readme.md` for how to run it. Please check the [Connecting to an Infotainment System](Getting Started/Connecting to an Infotainment System) guide for more detailed instructions on how to get the emulator's IP address and port number.
+This workaround for older versions of Core is also necessary for WebEngine apps.
+
+### Connecting as a WebSocket Server
+SDL Core acts as the WebSocket client in this case. The information about your app and how Core should connect to it must go into the policy table. Check the [Connecting to an Infotainment System](Getting Started/Connecting to an Infotainment System) guide's Configuring the Connection section for how to set up your policy table to point to your app
```

Comment:
```suggestion
SDL Core acts as the WebSocket client in this case. The information about your app and how Core should connect to it must go into the policy table. Check the [Connecting to an Infotainment System](Getting Started/Connecting to an Infotainment System) guide's **Configuring the Connection** section for how to set up your policy table to point to your app
```

That's usually how we highlight sections since we can't anchor link them for now.